control/src/lib.rs

```diff
@@ -452,11 +454,13 @@ pub mod pallet {
 		/// Emits `AddMember` event when successful.
 		///
 		/// Weight: `O(1)`
```

Comment:
This is actually my mistake. Since both `do_add_member` and `do_remove_member` contain binary search, it's not O(1) anymore.
```suggestion
		/// Weight: `O(log n)`
```